### PR TITLE
[Doc] Edit for grammar and style

### PR DESCRIPTION
**Pull request by john-andrilla**
As with other tutorials, it would help the reader if you add a paragraph in the opening section about assumptions or prerequisites.

Does this refer to SageMaker Ground Truth feature? "...assigns its ground truth label..." If yes, phrase it thus: assigns its Amazon SageMaker Ground Truth label

## Description
Edit voice and tone to improve readability.

### CODE REVIEW

**Comment by yzh119** [comment]
Hi John,
Thanks for your contribution.
In this tutorial "ground truth" refers to a term in machine learning and it's not related to SageMaker.

File: tutorials/basics/4_batch.py [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
 """
 
 ###############################################################################
-# Simple Graph Classification Task
+# Simple graph classification task
 # --------------------------------
-# In this tutorial, we will learn how to perform batched graph classification
-# with dgl via a toy example of classifying 8 types of regular graphs as below:
+# In this tutorial, you learn how to perform batched graph classification
+# with DGL. The example task objective is to classify eight types of regular graphs shown here.
```
Comment (jermainewang) [inline tutorials/basics/4_batch.py]: Actually, regular graph refers to a graph where all nodes have the same degree ([link](https://en.wikipedia.org/wiki/Regular_graph)). How about changing it to topology?

File: tutorials/basics/4_batch.py [hunk 2]
```diff
@@ -250,12 +252,12 @@ def forward(self, g):
     (test_Y == argmax_Y.float()).sum().item() / len(test_Y) * 100))
 
 ###############################################################################
-# Below is an animation where we plot graphs with the probability a trained model
-# assigns its ground truth label to it:
+# The figure here is an animation where you plot graphs with the probability that a trained model
+# assigns its Amazon SageMaker ground truth label to it.
```
Comment (jermainewang) [inline tutorials/basics/4_batch.py]: As @yzh119 has explained, the label is not from Amazon SageMaker. It is a categorical variable from the dataset itself indicating the graph type of each sample. How about "The animation here plots the probability a trained model predicts the correct graph type."?

File: tutorials/basics/4_batch.py (hunk 1 above)
Comment (john-andrilla) [reply tutorials/basics/4_batch.py]: @jermainewang Thanks for your comment. Can you clarify where to modify the newly proposed text?  This? The example task objective is to classify eight types of topologies shown here.

File: tutorials/basics/4_batch.py (hunk 2 above)
Comment (john-andrilla) [inline tutorials/basics/4_batch.py]: # The animation here plots the probability that a trained model predicts the correct graph type.

Comment (john-andrilla) [inline tutorials/basics/4_batch.py]:
```suggestion
```

File: tutorials/basics/4_batch.py [hunk 3]
```diff
@@ -250,12 +252,12 @@ def forward(self, g):
     (test_Y == argmax_Y.float()).sum().item() / len(test_Y) * 100))
 
 ###############################################################################
-# Below is an animation where we plot graphs with the probability a trained model
-# assigns its ground truth label to it:
+# The figure here is an animation where you plot graphs with the probability that a trained model
```
Comment (john-andrilla) [inline tutorials/basics/4_batch.py]: The animation here plots the probability that a trained model predicts the correct graph type.

Comment (john-andrilla) [inline tutorials/basics/4_batch.py]:
```suggestion
# The animation here plots the probability that a trained model predicts the correct graph type.
```

File: tutorials/basics/4_batch.py [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 
+In this tutorial, you learn how to use DGL to batch multiple graphs of variable size and shape. The 
+tutorial also demonstrates training a graph neural network for a simple graph classification task.
+
 Graph classification is an important problem
-with applications across many fields -- bioinformatics, chemoinformatics, social
-network analysis, urban computing and cyber-security. Applying graph neural
-networks to this problem has been a popular approach recently (
+with applications across many fields, such as bioinformatics, chemoinformatics, social
+network analysis, urban computing, and cyber-security. Applying graph neural
```
Comment (aaronmarkham) [inline tutorials/basics/4_batch.py]:
```suggestion
network analysis, urban computing, and cybersecurity. Applying graph neural
```

File: tutorials/basics/4_batch.py (hunk 1 above)
Comment (jermainewang) [reply tutorials/basics/4_batch.py]: Exactly yes!

Comment (john-andrilla) [inline tutorials/basics/4_batch.py]:
```suggestion
# with DGL. The example task objective is to classify eight types of topologies shown here.
```

**Pull request merged**